### DISCUSSION TITLE
Code actions for ltex-ls not working?

### DISCUSSION THREAD

**Discussion by abuturabofficial** [LazyVim/LazyVim#1847] [Q&A] [answered]
Apart from the 1st, other code actions doesn't work. It looks like I need to configure them, but honestly, after going through `ltex-ls` documentation, I couldn't comprehend how to do it properly. Anyone please help?

The most important one for me is the ability to `add to the dictionary`, and secondly `hide false positive`.

![image](https://github.com/LazyVim/LazyVim/assets/68259771/81079ce4-1d5e-4b05-8702-1a9695892288)

**Comment by mdietrich16** [comment, marked as answer]
I believe these actions are advertised by the server, but the client (Neovim) doesn't handle those. This is not a LazyVim specific issue, but a lspconfig one, which was discussed for example [here](https://github.com/neovim/nvim-lspconfig/issues/1443). However, the problem is only that lspconfig does not nor wants to provide off-spec support for these actions, you can implement them yourself or use [this plugin](https://github.com/vigoux/ltex-ls.nvim) for example. This adds the necessary "hooks" into the code action callbacks.
